add  login screen

kuldip0107 opened the pull request:
- today I created one login screen
- logo icon
- emai/password input
- foget password & create new accont button added
- below the screen shot the ourput
<img width="624" height="1028" alt="image" src="https://github.com/user-attachments/assets/701256e5-b8c8-4fbd-8f71-167a6efa63f7" />



https://github.com/user-attachments/assets/512674a9-f5eb-4c55-9062-47b5ec94c639

dr-bharada commented on src/components/screens/Login.tsx:
```diff
@@ -0,0 +1,175 @@
+
+ const Login: React.FC = () => {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```
Refactored to manage email & password in a single useState object – nice use of object state handling 👌

kuldip0107 commented:
https://github.com/user-attachments/assets/7e5de18e-9cb0-4043-aa8b-5856ea111cd7

- Added email and password validation with error messages
- Implemented show/hide password functionality using eye icon
- Disabled login button until inputs are valid
- Improved form UX by clearing errors while typing

dr-bharada commented on src/components/screens/Login.tsx:
```diff
@@ -0,0 +1,230 @@
+type FormErrors = Partial<FormData>;
+
+const FbLogin: React.FC = () => {
+  const [form, setForm] = useState<FormData>({ email: "", password: "" });
```
Used single `useState` for form data and errors - cleaner and easier to manage.

dr-bharada replied:
@kuldip0107  any update?

dr-bharada replied:
@kuldip0107

dr-bharada reviewed (commented):
Please resolve remaining comment

kuldip0107 commented:
<img width="486" height="1013" alt="Screenshot 2025-08-02 002156" src="https://github.com/user-attachments/assets/8e652ba1-0f4f-406b-861c-8a192ca47c0c" />

-refactor: combine form data & errors into single state and add background image
-feat(login): add background image and simplify state management using single useState
-ui: implemented background image & improved state handling with single useState
